admin_guide/scheduling/custom_scheduler.adoc

```diff
@@ -0,0 +1,102 @@
+[[admin-guide-sched-custom]]
+= Custom Schedulers
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+You can run multiple, custom schedulers alongside the default scheduler and configure which scheduler to use for each pods.
+
+To schedule a given pod using a specific scheduler, xref:admin-guide-sched-custom-deploy[specify the name of the scheduler in that pod spec].
+
+[[admin-guide-sched-custom-deploy]]
+== Deploying the Scheduler
+
+The steps below are the general process for deploying a scheduler into your cluster.
+
+[NOTE]
+====
+Information on how to create/deploy a scheduler is outside the scope of this document. For an example, see link:https://github.com/kubernetes/kubernetes/tree/master/plugin/pkg/scheduler[*_plugin/pkg/scheduler_* in the Kubernetes source directory].
+====
+
+. Create or edit a pod configuration and specify the name of the scheduler with the `schedulerName` parameter. The name must be unique.
++
+.Sample pod spec with scheduler
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: custom-scheduler
+  labels:
+    name: multischeduler-example
+spec:
+  schedulerName: custom-scheduler <1>
+  containers:
+  - name: pod-with-second-annotation-container
+    image: gcr.io/google_containers/pause:2.0
+----
```

Comment:
Callout formatting is wrong here, which is incorrectly rendering this doc. It should be:

====
----
Source
----
<1>
====